src/evaluators/utility/PHAL_Source_Def.hpp

```diff
@@ -17,6 +17,9 @@
 #include "Teuchos_Array.hpp"
 #include "Teuchos_TestForException.hpp"
 
+#include "PHAL_SharedParameter.hpp"
+#include "PHAL_Spatial.hpp"
```

Comment:
The name `PHAL_Spatial` is a bit obsucre. We could call it `PHAL_Field_Source` or `Source Functions`